src/getMachines.py

```diff
@@ -0,0 +1,51 @@
+import boto3
+from decEncoder import *
+
+"""Scan dynamo table"""
+dynamodb = boto3.resource('dynamodb', region_name = 'us-east-2')
+machineTable = dynamodb.Table('Machines')
+response = machineTable.scan()
+
+"""Lambda handler function for /buildings/{buildingId}/machines API call
+   Returns all the machines in a given building if no queries specified
+   If query (machineType and/or status) specified, filter machines matching those query parameters
+   If no buildingId or queries are specified, return all machines
```

Comment:
I'm not quite sure I understand why the buildingID wouldn't be specified in this API call. If anything, if the buildingID isn't specified we should return a 404. 

I think you might be thinking about the use case where someone wants to just get all of the machines. This would be a different API endpoint, and therefore a different lambda function. We can do that later if we want, someone might have a use case for it. But for now we don't need it, at least not that I can think of. If you can think of a reason, we can and that to the board.